x-pack/plugins/security_solution/scripts/quickstart/modules/data/index.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { range } from 'lodash';
+import crypto from 'crypto';
+
+export const buildLargeDocument = ({
```

Comment:
It could make small documents too, but I'd like to keep the name more specific to capture the intent and capabilities of the function so it's clear what use cases it would be most useful for. If I'm doing testing and just want any document, large or small, a function called `buildLargeDocument` might do just fine. But if I'm doing scale testing and want to make large documents and see a function called `buildDocument` I'd have to look more at the implementation to see what kind of document it builds.

Reply:
sounds good to me!